crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -191,13 +193,7 @@ impl RefreshTask {
             }
         };
 
-        match self.get_data_update(filters).await {
-            Ok(data) => Ok(data),
-            Err(e) => {
-                tracing::error!("Failed to load data for dataset {dataset_name}: {e}");
-                Err(e)
-            }
-        }
+        self.get_data_update(filters).await
```

Comment:
This is changed intentionally to fix duplicate error tracing bug: additional warning message after error message.